packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss

```diff
@@ -38,7 +38,6 @@
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
-  min-width: 100px;
```

Comment:
I'll set this to 30px. At that point any tags with less than ~4-5 characters will start being the same width, and the extra space on the rare tag of 1-2 characters still looks OK.